lochness/db/models/phoenix_files.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+"""
+File Model
+"""
+
+from pathlib import Path
+from datetime import datetime
+
+from lochness import db
+
+
+class PhoenixFiles:
+    """
+    Represents a file on the PHOENIX file system.
+
+    Attributes:
+        file_path (Path): The path to the file.
+    """
+
+    def __init__(self, file_path: Path, with_hash: bool = True):
+        """
+        Initialize a File object.
+
+        Automatically computes the file size and modification time.
+
+        Args:
+            file_path (Path): The path to the file.
+        """
+        self.file_path = file_path
+
+        if not file_path.exists():
+            raise FileNotFoundError(f"File not found: {file_path}")
+
+        self.file_name = file_path.name
+        self.file_type = file_path.suffix
```

Comment:
Can we also have modality variable, eg) phone, interview, mri, eeg and etc.?

Reply:
Since all PHOENIX files will have a corresponding Remote file (for its source), it will also have a row on the file mapping table, which will contain the modality information.

I can move that information to the `phoenix_files` table, if you that that is more appropriate!

Reply:
As long as we can easily filter by modality I agree with you.